charts/cosigned/templates/_helpers.tpl

```diff
@@ -116,6 +116,17 @@ Create the image path for the passed in image field
 {{- end -}}
 {{- end -}}
 
+{{/*
+Cosigned webhook name
+*/}}
+{{- define "cosigned.webhookName" -}}
+{{- if .Values.cosign.webhookName -}}
+{{- .Values.cosign.webhookName -}}
+{{- else -}}
+cosigned.sigstore.dev
```

Comment:
the values.yaml should set the default value, so you don't set it here in the helpers.

Reply:
Done